api/openapi-spec/swagger.json

```diff
@@ -226,6 +238,12 @@
       "401": {
        "description": "Unauthorized"
       }
+     },
+     "x-kubernetes-action": "LIST",
+     "x-kubernetes-group-version-kind": {
+      "Group": "",
+      "Version": "v1",
+      "Kind": "ConfigMap"
```

Comment:
The kind is not the response type, it is the resource Kind that this operation is for.  So I think the Kind is correct here (operation List, resource ConfigMap).  The response type is included in the non-extension portion of the API operation.